examples/sdk/node/src/index.ts

```diff
@@ -0,0 +1,67 @@
+import { BacktraceClient } from '@backtrace/node';
+import fs from 'fs';
+import path from 'path';
+import { exit } from 'process';
+import readline from 'readline';
+import { SUBMISSION_URL } from './consts';
+const reader = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout,
+});
+
+const client = BacktraceClient.builder({
+    url: SUBMISSION_URL,
+    attachments: [path.join(path.dirname(process.cwd()), 'samplefile.txt')],
+    rateLimit: 5,
+    userAttributes: {
+        'custom-attribute': 'test',
+        'custom-annotation': {
+            prop1: true,
+            prop2: 123,
+        },
+    },
+}).build();
+
+console.log(
+    `Welcome to the Backtrace demo. Please pick one of available options
+    1. Send an exception
+    2. Send a message
+
+    0. Exit`,
+);
+
+function showMenu() {
+    reader.question('Select the option... \n', async function executeUserOption(optionString: string) {
```

Comment:
Can we move the console log from lines 25-30 to this question? Make sure to also specify that you want the user to input a number.

Also, IMO it would be nicer if we would not add a newline and instead do something like this:
```
Select the option: <user input here>
```

Reply:
Moved it to the question. I also left the greeting message at the top of the menu, it sounds silly to show it every time.